website/docs/docs/dbt-versions/compatible-track-changelog.md

```diff
@@ -18,6 +18,48 @@ Starting in January 2025, each monthly "Extended" release will match the previou
 
 For more information, see [release tracks](/docs/dbt-versions/cloud-release-tracks).
 
+## January 2025
+
+Release date: January 14, 2025
+
+This release includes functionality from the following versions of dbt Core OSS:
+```
+dbt-core==1.9.1
+
+# shared interfaces
+dbt-adapters==1.13.1
+dbt-common==1.14.0
+dbt-semantic-interfaces==0.7.4
+
+# adapters
+dbt-athena==1.9.0
+dbt-bigquery==1.9.1
+dbt-databricks==1.9.1
+dbt-fabric==1.9.0
+dbt-postgres==1.9.0
+dbt-redshift==1.9.0
+dbt-snowflake==1.9.0
+dbt-spark==1.9.0
+dbt-synapse==1.8.2
```

Comment:
@emmyoop just double checking if synapse shoudl be 1.8.3?

![Screenshot 2025-01-14 at 18 19 08](https://github.com/user-attachments/assets/2642b19e-fa97-4235-8e8f-ad522dff2184)

Reply:
1.8.2 is the dbt-synapse versions that is in the current compatible.  dbt-synapse is just missing the changelogs for it, even though it is released.